### PR TITLE
README: add contribution and links

### PR DESCRIPTION
**Pull request by philips**
Ignition should be up to the standard of our other open source projects outlining how to get involved and where to find and file bugs. Follow the template of etcd.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,21 @@ Ignition is the utility used by CoreOS Container Linux to manipulate disks durin
...
+
+- Mailing list: [coreos-dev](https://groups.google.com/forum/?hl=en#!forum/coreos-dev)
+- IRC: #[coreos](irc://irc.freenode.org:6667/#coreos) on freenode.org
+- Bugs: [issues][issues]
+
+## Contributing
+
+See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
+
+To help triage or fix bugs see the current [ignition issues](https://github.com/coreos/bugs/labels/component%2Fignition).
```
Comment (crawford) [inline README.md]: "To help triage or fix bugs, see the..."

Ignition is also capitalized.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,21 @@ Ignition is the utility used by CoreOS Container Linux to manipulate disks durin
...
+
+## Reporting Bugs
+
+- To report a bug use the [bug tracker][issues]
```
Comment (crawford) [inline README.md]: "To report a bug, use the..."

File: README.md [hunk 3]
```diff
@@ -6,7 +6,21 @@ Ignition is the utility used by CoreOS Container Linux to manipulate disks durin
 
 Odds are good that you don't want to invoke Ignition directly. In fact, it isn't even present in the Container Linux root filesystem. Take a look at the [Getting Started Guide][getting started] for details on providing Ignition with a runtime configuration.
 
-Use the [bug tracker][issues] to report bugs.
+## Contact
+
+- Mailing list: [coreos-dev](https://groups.google.com/forum/?hl=en#!forum/coreos-dev)
+- IRC: #[coreos](irc://irc.freenode.org:6667/#coreos) on freenode.org
```
Comment (crawford) [inline README.md]: We recommend TLS when using IRC (port 6697).

File: README.md [hunk 4]
```diff
@@ -6,7 +6,21 @@ Ignition is the utility used by CoreOS Container Linux to manipulate disks durin
 
 Odds are good that you don't want to invoke Ignition directly. In fact, it isn't even present in the Container Linux root filesystem. Take a look at the [Getting Started Guide][getting started] for details on providing Ignition with a runtime configuration.
 
-Use the [bug tracker][issues] to report bugs.
+## Contact
+
+- Mailing list: [coreos-dev](https://groups.google.com/forum/?hl=en#!forum/coreos-dev)
```
Comment (crawford) [inline README.md]: Please use https://groups.google.com/forum/#!forum/coreos-dev instead. Not all users prefer English.

**Comment by ajeddeloh** [comment]
Lets finish this...

**Comment by dgonyeo** [comment]
ok to test

**Comment by dgonyeo** [comment]
@ajeddeloh feel free to +1 and merge on green

**Comment by ajeddeloh** [comment]
retest this please

**Comment by ajeddeloh** [comment]
Test failure is unrelated.

**Pull request merged**